database/models.py

```diff
@@ -11,10 +11,32 @@ class HrPartners(Base):
         """"""
         __tablename__ = 'hrpartners'
         __table_args__ = {'autoload': True}
+
+
+    class Applications(Base):
+        """"""
+        __tablename__ = 'applications'
+        __table_args__ = {'autoload': True}
+
+
+    class Users(Base):
+        """"""
+        __tablename__ = 'users'
+        __table_args__ = {'autoload': True}
```

Comment:
Co robi autoload? Nie zrobi nam to problemu n+1 zapytań?

Reply:
Autoload automatycznie ładuje klasę. Czyli ta klasa ma automatycznie parametry.